packages/ui/src/elements/Table/DefaultCell/fields/Relationship/index.tsx

```diff
@@ -34,18 +34,14 @@ export const RelationshipCell: React.FC<RelationshipCellProps> = ({
   const { documents, getRelationships } = useListRelationships()
   const [hasRequested, setHasRequested] = useState(false)
   const { i18n, t } = useTranslation()
-  const { searchParams, stringifyParams } = useSearchParams()
-
-  const stringParams = stringifyParams({ params: searchParams })
-
-  const previousParams = useRef(stringParams)
+  const { searchParams } = useSearchParams()
+  const [memoizedSearchParams, setMemoizedSearchParams] = useState(searchParams)
```

Comment:
`useRef` is more efficient than this because it does not trigger a re-render of the entire component